examples/emberjs/todomvc/README.md

```diff
@@ -7,9 +7,12 @@
 
 ## Note for people updating this app.
 
-The `index.html` and the `assets` folder in the parent folder as simlinks into the items with the
-same names inside `dist`. The `dist` folder has to be checked in git and built for production.
+The `index.html` and the `assets` folder in the parent folder as simlinks into
+the items with the same names inside `dist`. The `dist` folder has to be checked
+in git and built for production.
 
-To work on this comment `<base href="/examples/ember-cli/index.html" />` in the `app/index.html`
-and uncommented it back before doing the production build.
```

Comment:
I've removed this documentation, as I believe it is out of date.

Reply:
I think this README.md file should be moved to the root directory of the example

Reply:
Ah, I had the same feeling... I update the readme as Mixonic suggested and I move it to the root directory.